### PR TITLE
Provide a signer UI token by default.

### PR DESCRIPTION
**Pull request by gavofyork**

### CODE REVIEW

File: parity/main.rs [hunk 1]
```diff
@@ -186,6 +186,13 @@ fn execute_client(conf: Configuration, spec: Spec, client_config: ClientConfig)
 	let net_settings = conf.net_settings(&spec);
 	let sync_config = conf.sync_config(&spec);
 
+	// Create and display a new token for UIs.
+	if conf.args.flag_signer && !conf.args.flag_no_token {
```
Comment (tomusdrw) [inline parity/main.rs]: Tokens file will grow indefinitely :/

**Comment by gavofyork** [comment]
as long as people have "don't remember third party data" checked, like me, then it'll still grow, only also inconvenience the user while it does...

File: parity/main.rs (hunk 1 above)
Comment (gavofyork) [inline parity/main.rs]: as long as people have "don't remember third party data" checked, like me, then it'll still grow, only also inconvenience the user while it does...

Comment (tomusdrw) [inline parity/main.rs]: This particular setting should not affect neither extension nor minimal Signer UI webapp (can't reproduce it).
Perhaps you have some other setting checked like "Keep local data until you quit your browser".

That said, lgtm.

**Pull request merged**